app/helpers/lti_helper.rb

```diff
@@ -0,0 +1,115 @@
+require 'mono_logger'
```

Comment:
Where is this being used? Typically you don't need to explicitly require libraries under /app